src/Nethermind/Nethermind.Monitoring/Config/IMetricsConfig.cs

```diff
@@ -18,17 +18,20 @@
 
 namespace Nethermind.Monitoring.Config
 {
-    [ConfigCategory(Description = "Configuration of the Prometheus + Grafana metrics publication. Documentation of the required setup is not yet ready (but the metrics do work and are used by the dev team)")]
+    [ConfigCategory(Description = "Configuration of the Prometheus metrics publication. Documentation of the required setup is not yet ready (but the metrics do work and are used by the dev team)")]
```

Comment:
something is wrong with the commit -> MetricsServer is an empty file

Reply:
My bad, removed (The MetricsServer is just the one from the Prometheus package)